src/Symfony/Component/VarDumper/Tests/Test/VarDumpTestTraitTest.php

```diff
@@ -9,8 +9,32 @@
  * file that was distributed with this source code.
  */
 
-// Skipping trait tests for PHP < 5.4
-if (version_compare(PHP_VERSION, '5.4.0-dev', '>=')) {
-    require 'VarDumpTestTraitRequire54.php';
-}
+namespace Symfony\Component\VarDumper\Tests\Test;
+
+use Symfony\Component\VarDumper\Test\VarDumperTestTrait;
+
+class VarDumperTestTraitTest extends \PHPUnit_Framework_TestCase
```

Comment:
a mistake that was here before but could be fixed here: the name of the file is wrong (VarDump**er**)